lib/plottr_components/src/components/dialogs/ActsHelpModal.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import PropTypes from 'react-proptypes'
+import { Button, ButtonToolbar } from 'react-bootstrap'
+import { t } from 'plottr_locales'
+import UnconnectedPlottrModal from '../PlottrModal'
+
+const modalStyles = {
+  overlay: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  content: {
+    width: '75%',
+    position: 'relative',
+    left: 'auto',
+    bottom: 'auto',
+    right: 'auto',
+    minHeight: 500,
+    maxHeight: 'calc(100vh - 80px)',
+  },
+}
+
+const textStepsWrapper = {
```

Comment:
So we use an object for `modalStyles` because it's a convention set for the PlottrModal, but we use BEM for the other components.
I'd recommend creating an `scss` file with the name `acts_help_block.scss`.